pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -92,6 +92,9 @@ type PrometheusSpec struct {
 
 	// +optional
 	Config *PrometheusConfiguration `json:"config,omitempty"`
+
+	// +optional
+	Sidecars []corev1.Container `json:"sidecars,omitempty"`
```

Comment:
Plz change the name to `additionalContainers` as it is in https://github.com/pingcap/tidb-operator/pull/2229

Reply:
ok